### ISSUE TITLE
Has anyone train on other dataset successfully?

### ISSUE THREAD

**Issue by crh19970307** [matterport/Mask_RCNN#66]
I try the training code on my own dataset and the loss dosen't seem to go down whatever the learning rate is. Has anyone train on other dataset successfully? Give me some advice. Thx!

**Comment by mmejeras** [comment]
Do you mind letting me know how you trained on your own dataset? Are the instruction in Readme enough? I will provide my results once I've trained on  my own data.

**Comment by Chris-bupt** [comment]
Use CPU or GPU?

**Comment by waleedka** [comment]
There is a thread about training on the VOC dataset here https://github.com/matterport/Mask_RCNN/issues/51

Also, try reducing the learning rate and see if that helps. I just pushed an update to reduce the base learning rate as well, so get the latest code.

And, most importantly, check for bugs. Use the `inspect_data.ipynb` notebook to verify that your code is loading your dataset correctly.

**Comment by dexter1608** [comment]
@waleedka hi i wanted to ask you that when we load the dataset there is a function we call (self.add_image()) in which we give image id so my question is when I look  at the xml files of the dataset there are more than one object so add_image function is called for every object or for every image I am bit confused in this 
thank you

**Comment by waleedka** [comment]
@dexter1608 `add_image()` should be called once per image (not once per object). The IDs can be anything unique that you select to identify your images in the dataset.
